tests/nosetests/pyanaconda_tests/subscription_helpers_test.py

```diff
@@ -451,9 +461,13 @@ def register_password_missing_test(self, get_proxy, thread_mgr_wait, run_task):
         )
         # and the error callback should have been triggered
         error_callback.assert_called_once()
-        # int this case we fails before requesting any task and we should not
-        # attempt to run any
-        run_task.assert_not_called()
+        # in this case we fails before requesting any other task than
+        # the config one
+        subscription_proxy.SetRHSMConfigWithTask.assert_called_once()
+        run_task.assert_called()
+        # setting CDN as installation source does not make sense
+        # when we were not able to attach a subscription
+        set_cdn.assert_not_called()
```

Comment:
This variable is not defined.

Reply:
The variable was introduced by #2571, which has been just merged, so this should be good now.